### PR TITLE
fix: Edit get_block_hash syscall

### PR DESCRIPTION
**Pull request by stoobie**
### Description of the Changes

Added [discrete], other minor edits.

### PR Preview URL

https://starknet-io.github.io/starknet-docs/pr-909/documentation/architecture_and_concepts/Smart_Contracts/system-calls-cairo1/#get_block_hash

### Check List

- [x] Changes have been done against dev branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/909)

### CODE REVIEW

File: components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/system-calls-cairo1.adoc [hunk 1]
```diff
@@ -6,38 +6,40 @@ accessing the contract's storage, that standalone programs do not require. The S
...
 == `get_block_hash`
 
-
+[discrete]
 === Syntax
 
 [source,cairo,subs="+quotes,+macros"]
 ----
-fn get_block_hash(u64 blockNumber) -> felt252:
+fn get_block_hash(u64 block_number) -> felt252:
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/system-calls-cairo1.adoc:14]:
```suggestion
extern fn get_block_hash_syscall(
    block_number: u64
) -> SyscallResult<felt252> implicits(GasBuiltin, System) nopanic;
```

File: components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/system-calls-cairo1.adoc [hunk 2]
```diff
@@ -6,38 +6,40 @@ accessing the contract's storage, that standalone programs do not require. The S
...
 ----
 
+[discrete]
 === Description
 
-Gets the hash of a specific Starknet block within the range of `[first_v0_12_0_block, current_block-10]`.
-
+Gets the hash of a specific Starknet block within the range of `[first_v0_12_0_block, __current_block__ - 10]`.
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/system-calls-cairo1.adoc:20]:
```suggestion
Gets the hash of a specific Starknet block within the range of `[__first_v0_12_0_block__, __current_block__ - 10]`.
```

File: components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/system-calls-cairo1.adoc [hunk 3]
```diff
@@ -6,30 +6,37 @@ accessing the contract's storage, that standalone programs do not require. The S
...
 
-=== Error
-If `block_number` is greater than `current_block - 10`, an error is raised: `Block number out of range`
+link:https://github.com/starkware-libs/cairo/blob/0c882679fdb24a818cad19f2c18decbf6ef66153/corelib/src/starknet/syscalls.cairo#L37[`syscalls.cairo`^]
+
+[discrete]
+=== Error messages
 
-If `block_number` is less than the first block number of v0.12.0, the function returns zero.
+* *`Block number out of range`:* `block_number` is greater than `__current_block__ - 10`.
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/system-calls-cairo1.adoc:38]:
```suggestion
[horizontal,labelwidth="25",role="stripes-odd"]
`Block number out of range`:: `block_number` is greater than `__current_block__ - 10`.
```

File: components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/system-calls-cairo1.adoc [hunk 4]
```diff
@@ -6,30 +6,37 @@ accessing the contract's storage, that standalone programs do not require. The S
...
+* *`0`:* `block_number` is less than the first block number of v0.12.0.
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/system-calls-cairo1.adoc:39]:
```suggestion
`0`:: `block_number` is less than the first block number of v0.12.0.
```

**Comment by JameStark** [comment]
LGTM @stoobie

**Pull request merged**